### PR TITLE
Add Calculator Event Handler

### PR DESCRIPTION
**Pull request by Abdona**
Add event handler and bind it with the calculator buttons.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,34 +1,67 @@
+/* eslint-disable react/no-access-state-in-setstate */
+/* eslint-disable react/destructuring-assignment */
 import React from 'react';
+import calculate from '../logic/calculate';
```
Comment (vichuge) [inline src/components/Calculator.js:5]: - [ ] On importations for `Calculator.js` component need to be calculated and operate too, it looks like you put operate inside the calculate script in place this, please work on it. 🐶 

```
// ./src/components/Calculator.js

import React, { Component } from 'react';
import calculate from ../logic;
import operate from ../logic;
```

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Abdona) [reply src/components/Calculator.js:5]: The calculator.js file has operate function imported in it ,no need to import it again.

Comment (vichuge) [reply src/components/Calculator.js:5]: You don't need to import it again, but it's important to import it inside the `Calculator.js` in place of the `calculate.js` file, just like the example you have in my comment before. Please work on it.

Comment (vichuge) [reply src/components/Calculator.js:5]: - [ ] You don't need to import it again, but it's important to import it inside the `Calculator.js` in place of the `calculate.js` file, just like the example you have in my comment before, use the component to import is a better practice on React. Please work on it. 🐶

Comment (vichuge) [inline src/components/Calculator.js:5]: - [ ] You don't need to import it again, but it's important to import it inside the `Calculator.js` in place of the `calculate.js` file, just like the example you have in my comment before, use the component to import is a better practice on React. Please work on it. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ♻️ ⚠️ ⚠️ 🚦 🚦 

Hi dear student, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

star_wars_almost_there

## Aspects to improve ♻️
_Check the comments under the review._

- [ ] Check the comments under the review (an answer about what you say before this review, I just copy the explanation here). 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianotieno — APPROVED** [review]

**Review by christianotieno — CHANGES_REQUESTED** [review]
Hello <student name> 👋,

_This is an additional review as a result of the second opinion request._

Please ignore the earlier approval notice.
Since this is a second review opinion and having studied this matter closely, I would urge us to make the changes requested by @vichuge. From the review, you stated that you don't consider it necessary to import `operate.js` because that's inside `calculate.js`. But according to the guidelines, both should be inside `Calculator.js` component, not in the `calculate.js` logic. Please make this one final change before we proceed. Thank you!

Keep rocking 🚀! <can be customized>

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Abdona , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**